src/LightXML.jl

```diff
@@ -2,24 +2,17 @@ __precompile__()
 
 module LightXML
 
-using Compat; import Compat.String
-
-# iteratorsize is new in 0.5, declare it here for older versions. However,
-# we do not actually support calling these, since the traits are not defined
-if VERSION < v"0.5.0-dev+3305"
-    function iteratorsize(v)
-        error("Do not call this on older versions")
-    end
-else
-    import Base: iteratorsize, SizeUnknown, IsInfinite, HasLength
-end
+using Compat
 
+# We do not actually support calling these, since the traits are not defined
+import Base: SizeUnknown, IsInfinite, HasLength
+import Base: start, done, next, show, getindex, show, string, length
 
-if is_windows()
-    const libxml2 = Pkg.dir("WinRPM","deps","usr","$(Sys.ARCH)-w64-mingw32","sys-root","mingw","bin","libxml2-2")
-else
-    const libxml2 = "libxml2"
-end
+const libxml2 =
+    (Compat.Sys.iswindows()
+     ? Pkg.dir("WinRPM", "deps", "usr", "$(Sys.ARCH)-w64-mingw32", "sys-root", "mingw",
+               "bin", "libxml2-2")
+     : "libxml2")
```

Comment:
I think the switch  to the ternary operator makes this harder to read than:
```julia
const libxml2 = if Compat.Sys.iswindows()
    Pkg.dir("WinRPM", "deps", "usr", "$(Sys.ARCH)-w64-mingw32", "sys-root", "mingw", "bin",
        "libxml2-2")
else
    "libxml2"
end
```

Reply:
OK - I'm not used to using `if` as an expression!